webpack.config.js

```diff
@@ -0,0 +1,270 @@
+            : 'development',
+        devtool: isProduction
+            ? 'source-map'
+            : 'eval-cheap-source-map',
+        entry: getPath('src/index.tsx'),
+        node: false,
+        resolve: {
+            extensions: ['.js', '.jsx', '.ts', '.tsx'],
+            symlinks: false,
+        },
+        output: {
+            path: getPath('build/'),
+            publicPath: '/',
+            sourceMapFilename: '[file].map',
+            chunkFilename: isProduction
+                ? 'js/[name].[contenthash].chunk.js'
+                : 'js/[name].chunk.js',
+            filename: isProduction
+                ? 'js/[name].[contenthash].bundle.js'
+                : 'js/[name].bundle.js',
+            assetModuleFilename: isProduction
+                ? 'assets/[name].[contenthash][ext]'
+                : 'assets/[name][ext]',
+            clean: true,
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.[tj]sx?$/,
+                    include: getPath('src/'),
+                    exclude: /node_modules/,
+                    loader: 'babel-loader',
+                    options: {
+                        plugins: [
+                            !isProduction && require.resolve('react-refresh/babel'),
+                        ].filter(Boolean),
+                    },
+                },
+                {
+                    test: /\.css$/,
+                    include: getPath('src/'),
+                    exclude: /node_modules/,
+                    use: [
+                        isProduction ? MiniCssExtractPlugin.loader : 'style-loader',
+                        {
+                            loader: 'css-loader',
+                            options: {
+                                importLoaders: 1,
+                                modules: {
+                                    localIdentName: '[name]_[local]_[hash:base64:5]',
+                                    exportLocalsConvention: 'camelCaseOnly',
+                                },
+                                esModule: true,
+                                sourceMap: true,
+                            },
+                        },
+                        {
+                            loader: 'postcss-loader',
+                            ident: 'postcss',
+                            options: {
+                                sourceMap: true,
+                            },
+                        },
+                    ],
+                },
+                {
+                    test: /\.(eot|svg|ttf|woff|woff2|png|jpg|gif)$/i,
+                    exclude: /(node_modules)/,
+                    include: getPath('src/'),
+                    type: 'asset/resource',
+                    /*
+                    options: {
+                        name: isProduction
+                            ? 'assets/[name].[contenthash]][ext]',
+                            : 'assets/[name].[ext]',
```

Comment:
Please Remove unwanted comments

Reply:
The behavior of asset/resource is still not known, that's what commenting out the default configuration.
Will remove after assets are added and the behavior is confirmed.

Reply:
okay